tutorials/katacoda/4-eventing-intro-broker/step3.md

```diff
@@ -0,0 +1,77 @@
+## Publishing Events
+Finally, let's publish some events. The broker can only be accessed from within the cluster where Knative Eventing is installed. We will create a pod within that cluster to
+act as an event producer that will execute the curl commands.
+
+```
+kubectl apply -f - << EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    run: post-event
+  name: post-event
+spec:
+  containers:
+    # This could be any image that we can SSH into and has curl.
+  - image: radial/busyboxplus:curl
+    imagePullPolicy: IfNotPresent
+    name: post-event
+    resources: {}
+    stdin: true
+    terminationMessagePath: /dev/termination-log
+    terminationMessagePolicy: File
+    tty: true
+EOF
+# wait for it to be ready
+kubectl wait pods/post-event --for=condition=ContainersReady
+```{{execute}}
+
+Let us publish some [cloudevents](https://cloudevents.io/). The command below adds some cloud-events specific headers to the curl command. Publish red events using the command:
```

Comment:
add break line before last sentence.

Also code quotes for the colors `red` `blue` `orange`

Reply:
done!